python/TestHarness/schedulers/RunParallel.py

```diff
@@ -0,0 +1,130 @@
+from timeit import default_timer as clock
+
+from Tester import Tester
+from Scheduler import Scheduler
+from MooseObject import MooseObject
+from util import *
+import platform, time
+
+import os
+
+## This class provides an interface to run commands in parallel
+#
+# To use this class, call the .run() method with the tester options
+class RunParallel(Scheduler):
+    @staticmethod
+    def validParams():
+        params = Scheduler.validParams()
+        params.addRequiredParam('scheduler',    'RunParallel', "The name of this scheduler")
```

Comment:
Just get rid of it. You shouldn't need it at all. The inheritance from MooseObject should already do what you need to do. You can build any object without this parameter. I'd have to look at where you actually use this parameter.